packages/realm/src/schema/to-binding.ts

```diff
@@ -99,10 +99,16 @@ export function toBindingPropertySchema(name: string, schema: CanonicalPropertyS
   const result: BindingProperty = {
     name,
     type: toBindingPropertyType(schema),
-    isIndexed: schema.indexed,
     objectType: schema.objectType && schema.objectType in TYPE_MAPPINGS ? undefined : schema.objectType,
     linkOriginPropertyName: schema.property,
   };
+
+  if (schema.indexed === "fulltext") {
```

Comment:
Is `fulltext` a single word?  I would expect this to be either `full-text` or `fullText`.

Reply:
Usually it's written as "full-text". I didn't think too much about it, but I think it makes sense to write it like that